### PR TITLE
[Merged by Bors] - fix(Polynomial/Laurent): replace Lean 3 syntax in comments

### PR DESCRIPTION
**Pull request by grunweg**
---

I didn't test the syntax, corrections welcome :-).

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by grunweg** [comment]
CC @riccardobrasca @adomani I think you worked on this file; you might be interested.

**Comment by riccardobrasca** [comment]
I didn't really work on this file, but @adomani seems to be the author.

bors d+ [@adomani]

**Comment by riccardobrasca** [comment]
bors d-

**Comment by riccardobrasca** [comment]
bors d=[@adomani ]

**Comment by riccardobrasca** [comment]
bors d=[adomani]

**Comment by riccardobrasca** [comment]
It seems I don't remember the syntax to delegate to someone who is not the author of the PR, but I wanted to delegate @adomani

**Comment by riccardobrasca** [comment]
bors d=[@adomani]

**Comment by riccardobrasca** [comment]
bors d=adomani

File: Mathlib/Data/Polynomial/Laurent.lean [hunk 1]
```diff
@@ -56,12 +56,11 @@ Lots is missing!
...
---    refine (_ : R[ℕ] →[R] R[X]).comp _,
---    { exact ⟨(toFinsuppIso R).symm, by simp⟩ },
---    { refine ⟨λ r, comapDomain _ r (Set.injOn_of_injective (λ a b ab, Int.ofNat.inj ab) _), _⟩,
---      exact λ r f, comapDomain_smul _ _ _ }
---  end
+--    refine (_ : R[ℕ] →[R] R[X]).comp _
+--    · exact ⟨(toFinsuppIso R).symm, by simp⟩
+--    · refine ⟨fun r ↦ comapDomain _ r
+--        (Set.injOn_of_injective (fun a b ab ↦ Int.ofNat.inj ab) _), _⟩
+--      exact fun r f ↦ comapDomain_smul _ _ _
```
Comment (adomani) [inline Mathlib/Data/Polynomial/Laurent.lean:63]:
```suggestion
--   refine (?_ : R[ℕ] →[R] R[X]).comp ?_
--   · exact ⟨(toFinsuppIso R).symm, by simp⟩
--   · refine ⟨fun r ↦ comapDomain _ r
--       (Set.injOn_of_injective (fun _ _ ↦ Int.ofNat.inj) _), ?_⟩
--     exact fun r f ↦ comapDomain_smul ..
```

**Comment by adomani** [comment]
Thanks!  I verified the code and, except for missing `?`, it worked!

I pushed an update and will merge once it passes CI.

PS: the todo list refers to Riccardo, since he suggested those features, not because he is "responsible" for the file.

**Comment by grunweg** [comment]
Thank you for checking and the quick response.

**Pull request closed without merging**